### ISSUE TITLE
Must use import to load ES Module

### ISSUE THREAD

**Issue by bucko13** [paulmillr/noble-secp256k1#122]
When trying to use the library in some code to validate taproot addresses, I end up getting the error ` Must use import to load ES Module`  when using the following import and running jest tests

```
import * as ecc from "@noble/secp256k1"
```

I ran into the same thing when trying to use `tiny-secp256k1` and filed a [similar issue there](https://github.com/bitcoinjs/tiny-secp256k1/issues/135). I'm hoping maybe you might be familiar with a solution.

One update from that issue is that I've also tried with the following options passed to jest:

```
    "test:watch": "NODE_OPTIONS=--experimental-vm-modules npx jest --watch src",
```

**Comment by paulmillr** [comment]
this is not noble’s bug. Jest may not support ESM modules. or, it may not — with some additional flags. you will meed to investigate it

**Comment by bucko13** [comment]
Yep. Thanks. Jest configs is where I was able to find the answer. I wasn't experiencing this with any other module imports other than these secp256k1 libs (bitcoinjs-lib for example though imports no problem), so I was posting this not so much as a bug but rather more as a "has this been encountered before" issue. 

For anyone else that comes here for the same thing, here are the pieces that were needed to play nice with jest:

- the following in your jest config `preset: "ts-jest/presets/default-esm",`
- the following "prefix" to your npm script (or however you call jest): `NODE_OPTIONS=--experimental-vm-modules npx jest --watch sr`
- and the following in your package.json: `"type": "module",` (i.e. in the package you're importing the library into and testing in)
